install.bat

```diff
@@ -507,8 +540,18 @@ local function get_msvc_env_setup_cmd()
 	assert(type(vars.UNAME_M) == "string", "requires vars.UNAME_M to be set before calling this function.")
 	local x64 = vars.UNAME_M=="x86_64"
 
-	-- 1. try visual studio command line tools
-	local vcdir = get_visual_studio_directory()
+	-- 1. try visual studio command line tools of VS 2017 or higher
+	local vsdir = get_visual_studio_directory_from_vswhere()
+	if vsdir then
+		local vcvarsall = vsdir .. '\\VC\\Auxiliary\\Build\\vcvarsall.bat'
+		if exists(vcvarsall) then
+			local vcvarsall_args = { x86 = "", x86_64 = " x64" }
+			return ('call "%s"%s'):format(vcvarsall, vcvarsall_args[vars.UNAME_M])

```

Comment:
Could this be reachable with other architectures? (check if `vcvarsall_args[vars.UNAME_M]` is nil?)

Reply:
fixed by adding an assert before the return.